curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/6464a6d98c43acecb8a7867b.md

```diff
@@ -0,0 +1,202 @@
+---
+id: 6464a6d98c43acecb8a7867b
+title: Step 100
+challengeType: 0
+dashedName: step-100
+---
+
+# --description--
+
+Set the value of the `msg` property to the following string:
+
+```md
+decimalToBinary(1) returns "1" (base case) and gives that value to the stack below. Then it pops off the stack.
+```
+
+Note that, since the string itself contains double quotation marks, you will need to use single quotation marks for your string value.
```

Comment:
They could also use backticks or escape the quotes.

When testing this, probably best to look at the value of the object property:


```js
assert.equal(animationData[2].msg, 'decimalToBinary(1) returns "1" (base case) and gives that value to the stack below. Then it pops off the stack');
```

Reply:
I ended up doing something similar, but using `assert.match` with pretty permissive regex for these steps.